WordPress/gutenberg
# [RNMobile] RangeCell styling refactor

## Description

**NOTE**: That PR is based on `rnmobile/spacer`
Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1448
Ref to gb-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1511
Change `RangeCell` mobile component styling.

## How has this been tested?

1. Open ios or android app
2. Add `Spacer` component
3. Press settings button
4. Check `RangeCell` styling
## Screenshots 
ios dark mode unfocused | ios light mode unfocused
--- | ---
<img width="463" alt="Screenshot 2019-10-29 at 10 39 29" src="https://user-images.githubusercontent.com/22746080/67757751-ada89880-fa3c-11e9-8c52-e76e56da93e3.png"> | <img width="507" alt="Screenshot 2019-10-29 at 10 41 09" src="https://user-images.githubusercontent.com/22746080/67757775-b600d380-fa3c-11e9-81a1-8f392df9bd47.png">

ios dark mode focused | ios light mode focused
--- | ---
<img width="463" alt="Screenshot 2019-10-29 at 11 12 03" src="https://user-images.githubusercontent.com/22746080/67757897-f3fdf780-fa3c-11e9-80a1-d899a8c8bcea.png"> | <img width="507" alt="Screenshot 2019-10-29 at 10 41 14" src="https://user-images.githubusercontent.com/22746080/67757869-e0529100-fa3c-11e9-9011-0d607572b4c1.png">

android focused | android unfocused
--- | ---
<img width="427" alt="Screenshot 2019-10-29 at 10 44 07" src="https://user-images.githubusercontent.com/22746080/67757956-10019900-fa3d-11e9-9461-f29e0ccc306b.png"> | <img width="427" alt="Screenshot 2019-10-29 at 10 44 02" src="https://user-images.githubusercontent.com/22746080/67757971-16901080-fa3d-11e9-832c-7ec6273849e9.png">

* Reset Button functionality
<img src="https://user-images.githubusercontent.com/22746080/67758032-2e679480-fa3d-11e9-85ab-7e8696548d85.gif" width="300" />



## Types of changes

* add reset button
* move slider to new line
* change input styling






## Checklist:
